requirements/edx-sandbox/local.txt

```diff
@@ -1,13 +1,3 @@
-# DON'T JUST ADD NEW DEPENDENCIES!!!
-#
-# If you open a pull request that adds a new dependency, you should notify:
-#   * @edx/ospr - to check licensing
-#   * @edx/devops - to check system requirements
-
-# Install these packages from the edx-platform working tree
-# NOTE: if you change code in these packages, you MUST change the version
-# number in its setup.py or the code WILL NOT be installed during deploy.
-common/lib/calc
-common/lib/chem
-common/lib/sandbox-packages
-common/lib/symmath
+# Placeholder for code which hasn't yet been updated to no longer use this file.
```

Comment:
What code hasn't been updated? Ansible?

Reply:
Yes, a few different places in the configuration repo.  After that's updated I plan to come back and remove these now-unnecessary files.